app/forms/update_user_profile_form.rb

```diff
@@ -63,7 +59,7 @@ def set_attributes(params)
   end
 
   def attribute_taken?
-    @mobile_taken == true || @email_taken == true
+    email_taken?
   end
```

Comment:
We don't need this method anymore. The call to this method above on line 38 can be replaced with `!email_taken?`